packages/synapse-react-client/package.json

```diff
@@ -253,6 +253,7 @@
     "build:esbuild": "node esbuild.config.mjs",
     "build:docs": "pnpm storybook build",
     "build:gh-pages": "pnpm storybook build && touch storybook-static/.nojekyll",
+    "build-storybook": "pnpm storybook build",
```

Comment:
I removed this in a previous change because I assumed that chromatic would use the newer/preferred `storybook build` command, but it still relies on `build-storybook`